src/seedu/addressbook/data/person/Person.java

```diff
@@ -16,6 +16,8 @@
     private Address address;
 
     private final UniqueTagList tags;
+    private int sequenceNumber;
```

Comment:
You can create a getter function, to allow usage of `sequenceNumber`, since it is a private member.